addon/background.js

```diff
@@ -308,6 +308,39 @@ async function turnOffPrivateWarning() {
   await browser.storage.sync.set({hasSeenPrivateWarning});
 }
 
+let hasSentWidthEvent = false;
+
+async function increaseSidebarMaxWidth() {
+  if (browser.sideview !== undefined) {
```

Comment:
You’re already doing this check above on L163 and below on L361, so wouldn’t this always be truthy here, in which case is the `else..if` ever reachable?

Reply:
First of all, why are you awake? Second of all...this is meant to just be a PR for the acceptance criteria, but i borked it and brought in this stuff as well.